public/app/features/alerting/unified/components/silences/SilenceTableRow.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FC, Fragment, useState } from 'react';
+import { dateMath, GrafanaTheme, toDuration } from '@grafana/data';
+import { css } from '@emotion/css';
+import { Silence, SilenceMatcher } from 'app/plugins/datasource/alertmanager/types';
+import { AlertLabel } from '../AlertLabel';
+import { StateTag } from '../StateTag';
+import { CollapseToggle } from '../CollapseToggle';
+import { useRulesByMatcher } from '../../hooks/useRulesByMatcher';
+import { ActionButton } from '../rules/ActionButton';
+import { ActionIcon } from '../rules/ActionIcon';
+import { useStyles } from '@grafana/ui';
+import { RulesTable } from '../rules/RulesTable';
+
+interface Props {
+  className?: string;
+  silence: Silence;
+}
+
+const SilenceTableRow: FC<Props> = ({ silence, className }) => {
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(true);
+
+  const styles = useStyles(getStyles);
+  const { status, matchers, startsAt, endsAt, comment, createdBy } = silence;
+  const matchingRules = useRulesByMatcher(matchers as SilenceMatcher[]);
+
+  const dateDisplayFormat = 'YYYY-MM-DD HH:mm';
+  const startsAtDate = dateMath.parse(startsAt);
+  const endsAtDate = dateMath.parse(endsAt);
+  const duration = toDuration(endsAtDate?.diff(startsAtDate || '')).humanize();
+
+  return (
+    <Fragment>
+      <tr className={className}>
+        <td>
+          <CollapseToggle isCollapsed={isCollapsed} onToggle={(value) => setIsCollapsed(value)} />
+        </td>
+        <td>
+          <StateTag status={status.state}>{status.state}</StateTag>
+        </td>
+        <td className={styles.matchersCell}>
+          {matchers?.map(({ name, value }) => {
+            return <AlertLabel key={`${name}-${value}`} labelKey={name} value={value} />;
```

Comment:
maybe should show `=~` instead of `=` for regex matchers, to make it clear it's regex